galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -447,3 +447,16 @@ messages:
     args:
       - name: port
         type: string
+
+  - name: "GatewayNotDeclared"
+    code: IST0144
```

Comment:
I have used the same code in one of my [PR](https://github.com/istio/istio/pull/31993/files#diff-5ab445fd2493bb54c4667c7e74f7c0e5773ab35bf24fc5e639ab24d90a333274R452) which is still open. Probably I will have to use another code.

Reply:
oh sorry， i will change it to another one.